virttest/utils_netperf.py

```diff
@@ -166,6 +166,7 @@ def pack_compile(self, compile_option=""):
         self.env_cleanup(clean_all=False)
         cmd = "%s && %s " % (pre_setup_cmd, setup_cmd)
         try:
+            self.session.cmd("echo $?", timeout=60)
```

Comment:
Could you help explain how this change fix the failure？ I could not quite understand. Thanks a lot.

Reply:
It seems a very tricky fix. I accept as it works.